Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ubuntu:bionic as source
+
+# Set frontend mode as noninteractive (default answers to all questions)
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update &&\
+    apt-get upgrade -y && \
```

Comment:
apt-get upgrade in a Dockefile is discouraged because it results in a different outcome every time it is executed. It's better to rely on an up-to-date base image.

Reply:
> will remove it. :-)

> apt-get upgrade in a Dockefile is discouraged because it results in a different outcome every time it is executed. It's better to rely on an up-to-date base image.